src/org/zaproxy/zap/extension/pscanrulesAlpha/ZapAddOn.xml

```diff
@@ -9,6 +9,8 @@
 	<![CDATA[
 		Issue 2538: Strict-Transport-Security Header Scanner additional variants.<br>
 		Issue 2716: Implement a passive check for cookies without SameSite set.<br>
```

Comment:
Yes, both lines.
Btw, the "changes" entries should be short descriptions (the UI does not word wrap that).